### ISSUE TITLE
Input areas add extra characters when typing

### ISSUE THREAD

**Issue by enki1337** [thunder-app/thunder#1182]
### Bug Description

When trying to add my account, the input area to select my instance is buggy. It adds extra characters whenever I type anything. This is what I get when I type "hello":
![Screenshot_20240310-150008](https://github.com/thunder-app/thunder/assets/68851/83917f6a-12bf-4a69-8637-e2a23ad77853)

### Expected Behaviour

Input area contains only the text I typed.

### Steps to Reproduce

1. Go to add instance.
2. Type in instance box.

### App Version

0.2.9

### Device

Samsung s9+

### OS

Android 10

**Comment by hjiangsu** [comment]
Thanks for the report!

To help us reproduce the issue, could you let me know if you're using a custom keyboard? Additionally, can you test this out with the system keyboard to see if you're encountering the same issue? Thanks!

**Comment by enki1337** [comment]
Yes, it's a bit of an older keyboard, but this is the only app I've had trouble with. The system keyboard works as expected. The keyboard is called Kii.

**Comment by hjiangsu** [comment]
Thanks for the confirmation. Do you have a link to the keyboard (Kii)? I don't have a physical Android device to test with, but I can attempt to see if I can reproduce the issue using an emulator.

**Comment by enki1337** [comment]
I think I had an old apk from my previous phone that I installed from, so it may not be exactly the same version, but I found this via google:

https://www.apk4fun.com/apk/1051/

Sadly it hasn't been on the play store for several years.

**Comment by hjiangsu** [comment]
Unfortunately, I was not able to get Kii installed on an emulator (attempted using Android 9.0 and Android 10.0 versions). Without being able to reproduce the issue on my end, it'll be difficult to fix this issue.

[kii.webm](https://github.com/thunder-app/thunder/assets/30667958/46d77cd6-e46c-4d10-94d9-98c079fc2879)
